website/yarn.lock

```diff
@@ -13590,4 +13590,4 @@ yocto-queue@^0.1.0:
 zwitch@^1.0.0:
   version "1.0.5"
   resolved "https://registry.npmjs.org/zwitch/-/zwitch-1.0.5.tgz"
-  integrity sha512-V50KMwwzqJV0NpZIZFwfOD5/lyny3WlSzRiXgA0G7VUnRlqttta1L6UQIHzd6EuBY/cHGfwTIck7w1yH6Q5zUw==
+  integrity sha512-V50KMwwzqJV0NpZIZFwfOD5/lyny3WlSzRiXgA0G7VUnRlqttta1L6UQIHzd6EuBY/cHGfwTIck7w1yH6Q5zUw==
```

Comment:
Im not sure where this is coming from.. local docusaurus build ?

Reply:
The only change is a lacking new line at the end of the file (maybe due to OS differences on how the tool generates the the line ending at the end of the file or whatever?)

Reply:
I always running the docs live without building it.
Running live from your branch went fine

Reply:
since this file is not related to this PR, could we revert the changes in it before merge ?

Reply:
You got it !